.telemetryrc.json

```diff
@@ -10,5 +10,10 @@
     "output": "src/plugins/telemetry/schema/kbn_packages.json",
     "root": "packages/",
     "exclude": []
+  },
+  {
+    "output": "src/plugins/telemetry/schema/oss_platform.json",
```

Comment:
I see the configs above make a distinction between "plugins" and "packages", producing different output files.
Is it okay to not do the distinction here?

Reply:
I think so... IMO, the main reason we had to do distinction is that they had different roots.